release.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+RELEASE_DIR='release';
+
+if [ -z "$1" ]
+	then
+		echo "Error: Must pass version string (e.g. v1.2.3) as first parameter.";
+		exit 0;
+fi
+
+echo "Releasing version $1";
+
+if [ -d "$SCRIPT_DIR/release" ] && [ ! -d "$SCRIPT_DIR/release/.git" ]
+	then
+		# Cleanup and clone fresh copy of repo on build branch.
+		rm -r "$SCRIPT_DIR/$RELEASE_DIR";
+		git clone --recursive --branch build git@github.com:humanmade/hm-gutenberg-tools.git ${RELEASE_DIR}
+	else
+		# Cleanup any changes and reset to build branch.
+		cd "${SCRIPT_DIR}/${RELEASE_DIR}";
```

Comment:
Shellcheck wants us to make this (and other `cd` commands
```suggestion
		cd "${SCRIPT_DIR}/${RELEASE_DIR}" || exit;
```
so that the script halts if the `cd` fails. I'm not too fussed but thought I'd mention it.